### PR TITLE
verify genus 1 lists

### PR DESCRIPTION
**Pull request by BarinderBanwait**
Added the sage verification of ranks as discussed in this morning's call.

### CODE REVIEW

File: python_scripts/verify_positive_ranks_genus_1.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            rank_failures.append(d)
+
+    return rank_failures, sage_failures
+
+
+def main(directory="../positive_rank_lists/"):
+    
+    success = True
+    for G, location in [
+        (Gamma1(11), f"{directory}/11_list.json")
```
Comment (koffie) [inline python_scripts/verify_positive_ranks_genus_1.py]:
```suggestion
        (Gamma1(11), f"{directory}/11_list.json"),
```
Missing comma at the end of the line causing:

  File "/home/mderickx/quadratic-torsion/python_scripts/verify_positive_ranks_genus_1.py", line 43, in main
    (Gamma1(11), f"{directory}/11_list.json")
TypeError: 'tuple' object is not callable

File: python_scripts/verify_positive_ranks_genus_1.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    If you want to see debug information, run
+
+        sage verify_positive_ranks_genus_1.py --verbose
+
+"""
+
+import json
+import logging
+import argparse
+from sage.all import Gamma1
```
Comment (koffie) [inline python_scripts/verify_positive_ranks_genus_1.py]: Import 
```suggestion
from sage.all import Gamma0, Gamma1
```

Comment (koffie) [reply python_scripts/verify_positive_ranks_genus_1.py]: Missing import

File: python_scripts/verify_positive_ranks_genus_1.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        (Gamma1(14), f"{directory}/14_list.json"),
+        (Gamma1(15), f"{directory}/15_list.json"),
+        (Gamma0(20), f"{directory}/2_10_list.json"),
+        (Gamma0(24), f"{directory}/2_12_list.json"),
+    ]:
+
+        E = G.modular_abelian_variety().elliptic_curve()
+        with open(location, "r") as f:
+            d_vals = json.loads(f.read())
+        logging.info(f"Verifying ranks for {G} with d_vals: {d_vals}")
```
Comment (koffie) [inline python_scripts/verify_positive_ranks_genus_1.py]: Could you only print d_vals in debug mode. The output of the entire list is long and makes it unreadable.

```suggestion
        logging.info(f"Verifying ranks for {G}") 
        logging.debug(f"and twisting parameters: {d_vals}")
```

File: python_scripts/verify_positive_ranks_genus_1.py (hunk 1 above)
Comment (BarinderBanwait) [reply python_scripts/verify_positive_ranks_genus_1.py]: oops

File: python_scripts/verify_positive_ranks_genus_1.py (hunk 2 above)
Comment (BarinderBanwait) [reply python_scripts/verify_positive_ranks_genus_1.py]: oops

File: python_scripts/verify_positive_ranks_genus_1.py (hunk 3 above)
Comment (BarinderBanwait) [reply python_scripts/verify_positive_ranks_genus_1.py]: done

**Review by koffie — APPROVED** [review]

**Pull request merged**